deployment/E_heroku.md

```diff
@@ -1 +1,277 @@
-## FPO
+### What we'll need
+
+- a working application
+
+### Goals
+
+Our main goal for this is guide is to get a Phoenix application running on Heroku.
+
+## Steps
+
+- Sign up for Heroku and install the Toolbelt
+- Create the Heroku application
+- Add the Phoenix static buildpack
+- Make our project Heroku-ready
+- Deploy!
+
+## Sign up for Heroku and install the Toolbelt
+
+Signing up to Heroku is very simple, just head over to [https://signup.heroku.com/www-header](https://signup.heroku.com/www-header).
+
+Once this is done download the Heroku Toolbert installer for your platform at [https://toolbelt.heroku.com/](https://toolbelt.heroku.com/).
+
+This will allow you to access Heroku commands from your shell.
```

Comment:
If we could eliminate the "you" and "your" here, that would be ideal. Something like:

This will allow us to access Heroku commands from our shell.